src/main/java/Person.java

```diff
@@ -108,24 +108,36 @@ public void setFather(Person father) {
         this.father = father;
     }
 
-    public void addParents(){
```

Comment:
Had je expres de eerste commit met variabelen en methodes in de main-branch gezet? 

Voor de review is dat lastig, want ik zie nu bijv. je Pet klasse niet en kan daar geen comments bij zetten